tools/tdbg/dlq_v2_service.go

```
@@ -193,6 +193,10 @@ func (ac *DLQV2Service) ReadMessages(c *cli.Context) (err error) {
 			}
 			res, err := adminClient.GetDLQTasks(ctx, request)
 			if err != nil {
+				// If the DLQ does not exist yet, it's effectively empty, so we can safely return without an error.
+				if strings.Contains(err.Error(), "queue not found:") {
```

shall we return a notFound error from that api?

---

It is returning serviceerror.NotFound. But it could be other things(eg: table not found). I couldn't think about a better way to just check "queue not found".